RHEL/6/templates/csv/sysctl_values.csv

```diff
@@ -1,5 +1,6 @@
+# Add <sysctl_parameter_name, desired_value> to generate hard-coded OVAL and remediation content.
```

Comment:
In original `input/oval/templates/sysctl_values.csv` there used to be a comment that explained meaning of lines that have no parameter and end with a comma:

`# Add <sysctl_parameter_name,> to generate OVAL and remediation content that use the XCCDF value.`

I suggest moving it here as well, because it can be unclear why some lines have a number and some haven't.